synapse/lib/tags.py

```diff
@@ -34,7 +34,7 @@ def iterTagUp(tag,div='.'):
 
 def getTufoSubs(tufo, tag):
     '''
-    Return a list of tufo props for the given tag (and down).
+    Return a list of tufo prop subtractions for the given tag (and down).
```

Comment:
these are not subtractions :) just all the properties for a given tag and down that are present.  we happen to use them for subtraction in one instance, but the "subs" part of the name is about them being sub tags :)